src/components/WordGrid.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect } from "react"
+import { useWeldor } from "../hooks/useWeldor"
+import './WordGrid.styles.css'
+
+const Row = ({ word, savedWord, boxStatus, isActive }) => {
+    return (
+        <div className="container">
+            {Array.from({ length: 6 }, (_, i) => (
+                <div key={i} className={boxStatus[i] || 'letter-box'}>
+                    { isActive ? word[i] || '' : savedWord[i]}
+                </div>
+            ))}
+        </div>
+    )
+}
+
+export const WordGrid = () => {
+    const { 
+        guessedWord, 
+        guessedWordSet, 
+        boxStatusSet, 
+        activeBoxKey, 
+        winConditionMet,
+        loseConditionMet,
+        handleUserInput
+    } = useWeldor()
```

Comment:
This will cause a problem where we don't have a consistent useWeldor() instance across all of our components. Will fix on my branch.